### PR TITLE
Makes the wild shapeshift spell slightly less terrible

### PR DESCRIPTION
**Pull request by Incoming5643**
You can choose your form (once) now instead of getting random corgi-d. The forms themselves have been shuffled a bit for usefulness:
- Mouse (small hitbox, projectiles passing over, and vent crawling
- Chaos Magicarp (a riskier chaos staff, space movement)
- Ed209 (STUNS FOR DAYS YO)
- Juggernaut (Walling, wallpunching)
- Corgi (Meme, hats)

Moves wild shapeshift up to the generic, as all the generic was was a version that could only turn you into a goat.

:cl: incoming5643
rscadd: The wizard spell wild shapeshift has been made less terrible.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
i remember one round some guy traded tcs to be a chaos magicarp and killed like half the station and there was an admincomplaint about it
that's how you know this will be good

**Comment by Zilenan91** [comment]
>Ed-209

Oh my god please no it'll spit infinite tasers, stun batons, and cuffs forever.

**Comment by Incoming5643** [comment]
at the end of the day it's still just a bot, the AI can turn it off remotely.

**Comment by Zilenan91** [comment]
Not if it's a player, they don't give a shit.

**Comment by Incoming5643** [comment]
I'm pretty sure that bug was fixed

**Comment by YakumoChen** [comment]
EDs don't have a taser cooldown, just fyi. They can literally shoot taser shots as fast as they can click.

**Comment by Incoming5643** [comment]
They also can't actually harm anyone

**Comment by Zilenan91** [comment]
They can't harm anyone, you're right, but the wizard can if he shapeshifts out of it.

**Comment by KazeEspada** [comment]
:+1:  to everything but the bot.

:-1:  A bot? YOU CAN TURN INTO A ROBOT!

**Comment by Incoming5643** [comment]
IT'S MAGIC YOU CAN'T TELL ME WHAT TO DO MOM

File: code/datums/spells/shapeshift.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/effect/proc_holder/spell/targeted/shapeshift
-	name = "Shapeshift"
-	desc = "Change shapes"
+	name = "Wild Shapeshift"
+	desc = "Take on the shape of an animal for a time to use their natural abilities. Once you've made your choice it cannot be changed."
```
Comment (KazeEspada) [inline code/datums/spells/shapeshift.dm]: TAKE THE SHAPE OF AN ANIMAL! FUCKING ROBOT IS ON THE LIST?

File: code/datums/spells/shapeshift.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/effect/proc_holder/spell/targeted/shapeshift
-	name = "Shapeshift"
-	desc = "Change shapes"
+	name = "Wild Shapeshift"
```
Comment (KazeEspada) [inline code/datums/spells/shapeshift.dm]: If your gonna keep the robot change the name to: Shapechange.

http://www.d20srd.org/srd/spells/shapechange.htm
D&D memes.

**Comment by PKPenguin321** [comment]
>everybody is complaining about the ED and I'm just sitting here drooling over how strong chaos carp are

**Comment by Incoming5643** [comment]
Just as a clarifying statement here, this isn't a charitable kind of transform spell where you just get kicked back to human if you die transformed. Death is death. Things like chaos magicarp are balanced out by the fact that if you eat a paltry number of shots the round is over.

**Comment by Incoming5643** [comment]
Things should be squared away now

**Pull request merged**